src/app/tests/TestAttributeValueEncoder.cpp

```diff
@@ -18,10 +18,15 @@
 
 /**
  *    @file
- *      This file implements unit tests for CommandPathParams
+ *      This file implements unit tests for AttributeValueEncoder
```

Comment:
nit: instead of fixing I would consider dropping this entire comment block: it often has negative benefit:

- it is obvious `a file named testAttributeValueEncoder obviously tessts the AttributeValueEncoder`
- tests are copied and pasted for license and header include boilerplate ... and then this comment becomes copy and pasted and sometimes not updated, making us spend time to update it.

Reply:
totally Agree, this same exact comment is already in half the tests